src/main/java/in/twizmwaz/cardinal/module/modules/titleRespawn/TitleRespawn.java

```diff
@@ -0,0 +1,459 @@
+            if (timeLeftToRespawn > 0) {
+                if (hasLeftClicked(id) || this.auto) {
+                    player.setSubtitle(TextComponent.fromLegacyText(ChatColor.GREEN + new LocalizedChatMessage(ChatConstant.UI_RESPAWN_AUTO, ChatColor.AQUA + formatter.format(timeLeftToRespawn).replace(",", ".") + ChatColor.GREEN).getMessage(player.getLocale())));
+                } else {
+                    player.setSubtitle(TextComponent.fromLegacyText(ChatColor.GREEN + new LocalizedChatMessage(ChatConstant.UI_RESPAWN_SCHEDULE, ChatColor.AQUA + formatter.format(timeLeftToRespawn).replace(",",".") + ChatColor.GREEN).getMessage(player.getLocale())));
+                }
+            } else if (isAllowedToRespawn(id)) {
+                if (hasLeftClicked(id) || this.auto) {
+                    respawnPlayer(player);
+                /*} else if (this.message != null) {
+                    getSubtitleMessage();*/
+                } else {
+                    player.setSubtitle(TextComponent.fromLegacyText(ChatColor.GREEN + new LocalizedChatMessage(ChatConstant.UI_RESPAWN_CLICK).getMessage(player.getLocale())));
+                }
+            }
+        }
+        return null;
+    }
+
+    //TODO Begin work on message attribute - localization
+    public String getSubtitleMessage() {
+        return this.message;
+    }
+
+    private void dump(Player player) {
+        for (ItemStack stack : player.getInventory().getArmorContents()) {
+            if (stack != null && stack.getType() != Material.AIR) {
+                player.getWorld().dropItemNaturally(player.getLocation().add(0, 0.5, 0), stack);
+            }
+        }
+        for (int i = 0; i < 36; i++) {
+            ItemStack stack = player.getInventory().getItem(i);
+            if (stack != null && stack.getType() != Material.AIR) {
+                player.getWorld().dropItemNaturally(player.getLocation().add(0, 0.5, 0), stack);
+            }
+        }
+        player.getInventory().clear();
+        player.getInventory().setArmorContents(null);
+    }
+
+
+
+
+
+    public void sendBlood(Player player) {
+        sendWorldBorderPacket(player, 1000000000);
+    }
+
+    public void removeBlood(Player player) {
+        sendWorldBorderPacket(player, 10);
+    }
+
+    protected void sendWorldBorderPacket(Player player, int warningBlocks) {
+        EntityPlayer nmsPlayer = ((CraftPlayer) player).getHandle();
+        WorldBorder playerWorldBorder = nmsPlayer.world.getWorldBorder();
+        PacketPlayOutWorldBorder worldBorder = new PacketPlayOutWorldBorder(playerWorldBorder, PacketPlayOutWorldBorder.EnumWorldBorderAction.SET_WARNING_BLOCKS);
+
+        try {
+            Field field = worldBorder.getClass().getDeclaredField("b");
+            field.setAccessible(true);
+
+            //TODO Add fields - http://wiki.vg/Protocol#World_Border
+
+            field.setInt(worldBorder, warningBlocks);
```

Comment:
Wonder if somebody knows what's wrong here. I don't get errors on the console, the red border just does not work.